usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -248,30 +232,25 @@ def create_place_of_performance_object(db_row_dict):
     )
 
 
-def create_officers_object(award, mapper, transaction_type):
-
-    transaction = fetch_latest_ec_details(award["id"], mapper, transaction_type)
-
-    officers = []
-
-    if transaction:
-        for officer_num in range(1, 6):
-            officer_name_key = "_officer_{}_name".format(officer_num)
-            officer_amount_key = "_officer_{}_amount".format(officer_num)
-            officer_name = transaction.get(officer_name_key)
-            officer_amount = transaction.get(officer_amount_key)
-            if officer_name or officer_amount:
```

Comment:
It didn't strike me as beneficial to only provide officer details if they were not null. Removed conditional to the response object always has 5 nested objects of `name` and `amount`

Reply:
I would agree in terms of complying with DAIMS, but that means returning keys with null values in the response body. I personally like showing the nulls, but the loudest voices in REST disagree: 
https://softwareengineering.stackexchange.com/questions/285010/null-vs-missing-key-in-rest-api-response

Reply:
Other devs I talk to almost unanimously prefer `null` rather than missing keys.

![image](https://user-images.githubusercontent.com/22781949/65256624-0d666800-dabd-11e9-94d2-00069912ca84.png)

Reply:
Awesome. Count me among them.